doc/conf.py

```diff
@@ -44,13 +44,18 @@
 numpydoc_class_members_toctree = False
 
 
-# pngmath / imgmath compatibility layer for different sphinx versions
-import sphinx
-from distutils.version import LooseVersion
-if LooseVersion(sphinx.__version__) < LooseVersion('1.4'):
-    extensions.append('sphinx.ext.pngmath')
+# math compatibility layer for different sphinx versions and settings
+if os.environ.get('USE_MATHJAX'):
+    extensions.append('sphinx.ext.mathjax')
+    mathjax_path = ('https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/'
+                    'MathJax.js?config=TeX-AMS_SVG')
 else:
-    extensions.append('sphinx.ext.imgmath')
+    import sphinx
+    from distutils.version import LooseVersion
+    if LooseVersion(sphinx.__version__) < LooseVersion('1.4'):
```

Comment:
I don't think we should bother with sphinx < 1.4, IIRC sphinx-gallery needs sphinx >= 1.5 anyway.